src/kem/saber/CMakeLists.txt

```diff
@@ -20,6 +20,14 @@ if(OQS_ENABLE_KEM_saber_lightsaber_avx2)
     set(_SABER_OBJS ${_SABER_OBJS} $<TARGET_OBJECTS:saber_lightsaber_avx2>)
 endif()
 
+if(OQS_ENABLE_KEM_saber_lightsaber_aarch64)
```

Comment:
Why are we getting Saber changes in this PR?

Reply:
This likely happened when I ran copy_from_upstream. I'll revert this.

Reply:
> Why are we getting Saber changes in this PR?
> This likely happened when I ran copy_from_upstream. I'll revert this.

Would it be a problem also updating Saber (or anything else that got upgraded in PQClean) at the same time? Shouldn't our CI (or doc generation) tell us if such "automatic" upgrade would be bad?